### ISSUE TITLE
Hangs on "checking" intermittently

### ISSUE THREAD

**Issue by AlexLovato** [emolchanov/npm-svn#5]
when installing an svn dependency it will hang on "Checking <dependency name>"... is there a way to catch when errors happen and alert the user?

**Comment by emolchanov** [comment]
yep, I'm planning new version of npm-svn with tests and errors handling in short time.
